cmd/deployment/create.go

```diff
@@ -184,28 +218,28 @@ func initFlags() {
 
 	createCmd.Flags().String("es-ref-id", "main-elasticsearch", "Optional RefId for the Elasticsearch deployment")
 	createCmd.Flags().Int32("es-zones", 1, "Number of zones the Elasticsearch instances will span")
-	createCmd.Flags().Int32("es-size", 4096, "Memory (RAM) in MB that each of the Elasticsearch instances will have")
+	createCmd.Flags().String("es-size", "4g", "Memory (RAM) in MB that each of the Elasticsearch instances will have")
```

Comment:
We need to change this to word that the allowed size is in "gb" or "g" notation
```suggestion
	createCmd.Flags().String("es-size", "4g", "Memory (RAM) in GB that each of the Elasticsearch instances will have")
```